abuild.in

```diff
@@ -214,7 +214,7 @@ default_sanitycheck() {
 		while [ $# -gt 1 ]; do
 			local file="$2"
 			shift 2
-			source_has $file || die "$file exists in ${algo}sums but is missing in source"
+			source_has $file || die "$file exists in ${algo}sums but is missing in the source variable"
```

Comment:
What about just using \$source?

Reply:
I've just changed it, thanks for the feedback!